File: demo07.pl
#!/usr/bin/perl -w



print "enter a number", "\n";
$num = <STDIN>;

if ($num == 0) {
    print "1", "\n";
} else {
    $x = 0;
    $y = 1;
    foreach $i (1..$num - 1) {
        $z = $x + $y;
        $x = $y;
        $y = $z;
    }
    print $x , "\n";
}

File: demo07.py
#!/bin/usr/python

#Calculate the nth fibonacci number iteratively
#Written by Raphael Rueda

import sys

print "enter a number"
num = int(sys.stdin.readline())

if num == 0: 
    print "1"
else:
    x = 0
    y = 1
    for i in range(1, num):
	z = x + y
	x = y
	y = z
    print x 
